csi/deployment_util.go

```diff
@@ -204,7 +212,26 @@ func CheckMountPropagationWithNode(managerURL string) error {
 	}
 	nodeCollection, err := apiClient.Node.List(&longhornclient.ListOpts{})
 	for _, node := range nodeCollection.Data {
-		if !node.MountPropagation {
+		retryCount := 0
+		conditions := node.Conditions
+		condition := conditions[string(types.NodeConditionTypeMountPropagation)].(map[string]interface{})
+		for {
```

Comment:
It's weird to use an infinite loop for retry here. Just use a normal for loop. You can check if the condition is still unknown outside the loop.